test/e2e/test_worker_status.py

```diff
@@ -17,31 +17,6 @@
 
 @pytest.mark.parametrize("operating_system", [os.environ["OPERATING_SYSTEM"]], indirect=True)
 class TestWorkerStatus:
-    def test_worker_lifecycle_status_is_expected(
```

Comment:
This seems not related to the PR?

Reply:
Was noted with the PR that this is moved to reduce the number of workers needed.